packages/block-library/package.json

```
@@ -27,10 +27,12 @@
 	"sideEffects": [
 		"build-style/**",
 		"src/**/*.scss",
-		"{src,build,build-module}/*/init.js"
+		"{src,build,build-module}/*/init.js",
+		"src/utils/interactivity/index.js"
```

This file is consume through `src` only by React Native, web bundlers would mostly use `build-module`:
```suggestion
		"{src,build,build-module}/utils/interactivity/index.js"
```